storm:2.6.2-jre17 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_storm:2.6.2-jre17_2024-06-12_20:22:07.json storm:2.6.2-jre17] execution failed: exit status 1
Output: 2024-06-12T20:22:08Z	INFO	Need to update DB
2024-06-12T20:22:08Z	INFO	Downloading DB...	repository="ghcr.io/aquasecurity/trivy-db:2"
31.53 MiB / 48.04 MiB [------------------------------------------------------------------------------>________________________________________] 65.63% ? p/s ?48.04 MiB / 48.04 MiB [--------------------------------------------------------------------------------------------------------------------->] 100.00% ? p/s ?48.04 MiB / 48.04 MiB [--------------------------------------------------------------------------------------------------------------------->] 100.00% ? p/s ?48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 27.52 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 27.52 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 27.52 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 25.74 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 25.74 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 25.74 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 24.08 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 24.08 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 24.08 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-----------------------------------------------------------------------------------------------------------] 100.00% 20.79 MiB p/s 2.5s2024-06-12T20:22:11Z	INFO	Vulnerability scanning is enabled
2024-06-12T20:22:11Z	FATAL	Fatal error	image scan error: scan error: scan failed: failed analysis: analyze error: pipeline error: failed to analyze layer (sha256:014e127d44a8ab0c5a8fe7f05f0aa778a0d3e36f55796ec708e422ab06f70a39): unable to get uncompressed layer sha256:014e127d44a8ab0c5a8fe7f05f0aa778a0d3e36f55796ec708e422ab06f70a39: failed to get the layer (sha256:014e127d44a8ab0c5a8fe7f05f0aa778a0d3e36f55796ec708e422ab06f70a39): unable to populate: unable to open: failed to copy the image: image "storm:2.6.2-jre17": not found
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
storm:2.6.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
storm:2.6-jre17 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_storm:2.6-jre17_2024-06-12_20:23:48.json storm:2.6-jre17] execution failed: exit status 1
Output: 2024-06-12T20:23:49Z	INFO	Vulnerability scanning is enabled
2024-06-12T20:23:49Z	FATAL	Fatal error	image scan error: scan error: scan failed: failed analysis: analyze error: pipeline error: failed to analyze layer (sha256:a6ae70336728cb7a3357a93599e2fef6bd7450357dbf182f8df9a21613396444): unable to get uncompressed layer sha256:a6ae70336728cb7a3357a93599e2fef6bd7450357dbf182f8df9a21613396444: failed to get the layer (sha256:a6ae70336728cb7a3357a93599e2fef6bd7450357dbf182f8df9a21613396444): unable to populate: unable to open: failed to copy the image: image "storm:2.6-jre17": not found
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
storm:2.6 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
storm:2.6.1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
plone:python38 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
plone:5.2.13-python38 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
plone:5.2.13 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
plone:5.2-python38 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
plone:5.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
bash:devel-alpine3.20 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
bash:devel-20240603-alpine3.20 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
bash:devel-20240603 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
bash:devel ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
bash:4.0.44-alpine3.20 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
amazonlinux:2023.4.20240528.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
amazonlinux:2023 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
amazonlinux:2.0.20240529.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
amazonlinux:2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
amazonlinux:2023.4.20240416.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
composer:2.4.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
composer:2.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
composer:2.3.10 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
composer:2.3 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
composer:2.2.23 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-mosquitto:openssl ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
eclipse-mosquitto:2.0.18-openssl ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-mosquitto:2.0-openssl ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-mosquitto:2-openssl ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-mosquitto:1.6.15-openssl ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
nextcloud:stable-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
nextcloud:stable-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
nextcloud:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
nextcloud:production-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
nextcloud:production-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
silverpeas:6.3.5 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
silverpeas:6.3.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
silverpeas:6.0.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
silverpeas:6.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
silverpeas:6.1.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swift:slim ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swift:rhel-ubi9-slim ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
swift:rhel-ubi9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swift:noble ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
swift:mantic ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
haxe:4.3.4-windowsservercore-ltsc2022 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_haxe:4.3.4-windowsservercore-ltsc2022_2024-06-12_21:06:22.json haxe:4.3.4-windowsservercore-ltsc2022] execution failed: exit status 1
Output: 2024-06-12T21:06:23Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:06:23Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "haxe:4.3.4-windowsservercore-ltsc2022" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (haxe:4.3.4-windowsservercore-ltsc2022): Error response from daemon: No such image: haxe:4.3.4-windowsservercore-ltsc2022
	* containerd error: image not found in containerd store: haxe:4.3.4-windowsservercore-ltsc2022
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: no child with platform linux/amd64 in index haxe:4.3.4-windowsservercore-ltsc2022


 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new8/Grype_scan_haxe:4.3.4-windowsservercore-ltsc2022_2024-06-12_21:06:22.json haxe:4.3.4-windowsservercore-ltsc2022] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'haxe:4.3.4-windowsservercore-ltsc2022':
  - docker: unable to inspect existing image: Error response from daemon: No such image: haxe:4.3.4-windowsservercore-ltsc2022
  - podman: podman not available: no host address
  - containerd: pull failed: no match for platform in manifest: not found
  - oci-registry: failed to get image from registry: no child with platform linux/amd64 in index haxe:4.3.4-windowsservercore-ltsc2022
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test haxe:4.3.4-windowsservercore-ltsc2022 --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new8/Snyk_scan_haxe:4.3.4-windowsservercore-ltsc2022_2024-06-12_21:06:22.json] execution failed: exit status 2
Output: The image does not exist for the current platform
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
haxe:4.3.4-windowsservercore-1809 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_haxe:4.3.4-windowsservercore-1809_2024-06-12_21:09:28.json haxe:4.3.4-windowsservercore-1809] execution failed: exit status 1
Output: 2024-06-12T21:09:28Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:09:29Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "haxe:4.3.4-windowsservercore-1809" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (haxe:4.3.4-windowsservercore-1809): Error response from daemon: No such image: haxe:4.3.4-windowsservercore-1809
	* containerd error: image not found in containerd store: haxe:4.3.4-windowsservercore-1809
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: no child with platform linux/amd64 in index haxe:4.3.4-windowsservercore-1809


 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new8/Grype_scan_haxe:4.3.4-windowsservercore-1809_2024-06-12_21:09:28.json haxe:4.3.4-windowsservercore-1809] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'haxe:4.3.4-windowsservercore-1809':
  - docker: unable to inspect existing image: Error response from daemon: No such image: haxe:4.3.4-windowsservercore-1809
  - podman: podman not available: no host address
  - containerd: pull failed: no match for platform in manifest: not found
  - oci-registry: failed to get image descriptor from registry: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4-windowsservercore-1809: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_haxe:4.3.4-windowsservercore-1809_2024-06-12_21:09:28.json haxe:4.3.4-windowsservercore-1809] execution failed: exit status 1
Output:     ! No image found for platform linux/amd64 trying to pull the image for platform windows/amd64:10.0.17763.5936
    ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image haxe:4.3.4-windowsservercore-1809: failed to pull image haxe:4.3.4-windowsservercore-1809: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4-windowsservercore-1809: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test haxe:4.3.4-windowsservercore-1809 --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new8/Snyk_scan_haxe:4.3.4-windowsservercore-1809_2024-06-12_21:09:28.json] execution failed: exit status 2
Output: no supported manifest found for platform: {"os":"linux","architecture":"amd64"}

haxe:4.3.4-windowsservercore ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_haxe:4.3.4-windowsservercore_2024-06-12_21:09:36.json haxe:4.3.4-windowsservercore] execution failed: exit status 1
Output: 2024-06-12T21:09:36Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:09:37Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "haxe:4.3.4-windowsservercore" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (haxe:4.3.4-windowsservercore): Error response from daemon: No such image: haxe:4.3.4-windowsservercore
	* containerd error: image not found in containerd store: haxe:4.3.4-windowsservercore
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4-windowsservercore: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_haxe:4.3.4-windowsservercore_2024-06-12_21:09:36.json haxe:4.3.4-windowsservercore] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image haxe:4.3.4-windowsservercore: failed to pull image haxe:4.3.4-windowsservercore: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4-windowsservercore: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new8/Grype_scan_haxe:4.3.4-windowsservercore_2024-06-12_21:09:36.json haxe:4.3.4-windowsservercore] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'haxe:4.3.4-windowsservercore':
  - docker: pull failed: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - podman: podman not available: no host address
  - containerd: pull failed: no match for platform in manifest: not found
  - oci-registry: failed to get image descriptor from registry: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4-windowsservercore: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test haxe:4.3.4-windowsservercore --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new8/Snyk_scan_haxe:4.3.4-windowsservercore_2024-06-12_21:09:36.json] execution failed: exit status 2
Output: no supported manifest found for platform: {"os":"linux","architecture":"amd64"}

haxe:4.3.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_haxe:4.3.4_2024-06-12_21:09:44.json haxe:4.3.4] execution failed: exit status 1
Output: 2024-06-12T21:09:45Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:09:45Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "haxe:4.3.4" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (haxe:4.3.4): Error response from daemon: No such image: haxe:4.3.4
	* containerd error: image not found in containerd store: haxe:4.3.4
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_haxe:4.3.4_2024-06-12_21:09:44.json haxe:4.3.4] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image haxe:4.3.4: failed to pull image haxe:4.3.4: GET https://index.docker.io/v2/library/haxe/manifests/4.3.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
haxe:4.3-windowsservercore-ltsc2022 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_haxe:4.3-windowsservercore-ltsc2022_2024-06-12_21:10:07.json haxe:4.3-windowsservercore-ltsc2022] execution failed: exit status 1
Output: 2024-06-12T21:10:08Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:10:09Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "haxe:4.3-windowsservercore-ltsc2022" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (haxe:4.3-windowsservercore-ltsc2022): Error response from daemon: No such image: haxe:4.3-windowsservercore-ltsc2022
	* containerd error: image not found in containerd store: haxe:4.3-windowsservercore-ltsc2022
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/haxe/manifests/4.3-windowsservercore-ltsc2022: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new8/Grype_scan_haxe:4.3-windowsservercore-ltsc2022_2024-06-12_21:10:07.json haxe:4.3-windowsservercore-ltsc2022] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'haxe:4.3-windowsservercore-ltsc2022':
  - docker: unable to inspect existing image: Error response from daemon: No such image: haxe:4.3-windowsservercore-ltsc2022
  - podman: podman not available: no host address
  - containerd: pull failed: no match for platform in manifest: not found
  - oci-registry: failed to get image descriptor from registry: GET https://index.docker.io/v2/library/haxe/manifests/4.3-windowsservercore-ltsc2022: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_haxe:4.3-windowsservercore-ltsc2022_2024-06-12_21:10:07.json haxe:4.3-windowsservercore-ltsc2022] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image haxe:4.3-windowsservercore-ltsc2022: failed to pull image haxe:4.3-windowsservercore-ltsc2022: GET https://index.docker.io/v2/library/haxe/manifests/4.3-windowsservercore-ltsc2022: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test haxe:4.3-windowsservercore-ltsc2022 --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new8/Snyk_scan_haxe:4.3-windowsservercore-ltsc2022_2024-06-12_21:10:07.json] execution failed: exit status 2
Output: The image does not exist for the current platform

adminer:standalone ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_adminer:standalone_2024-06-12_21:10:15.json adminer:standalone] execution failed: exit status 1
Output: 2024-06-12T21:10:16Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:10:17Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "adminer:standalone" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (adminer:standalone): Error response from daemon: No such image: adminer:standalone
	* containerd error: image not found in containerd store: adminer:standalone
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/adminer/manifests/standalone: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_adminer:standalone_2024-06-12_21:10:15.json adminer:standalone] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image adminer:standalone: failed to pull image adminer:standalone: GET https://index.docker.io/v2/library/adminer/manifests/standalone: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
adminer:fastcgi ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_adminer:fastcgi_2024-06-12_21:10:34.json adminer:fastcgi] execution failed: exit status 1
Output: 2024-06-12T21:10:35Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:10:36Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "adminer:fastcgi" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (adminer:fastcgi): Error response from daemon: No such image: adminer:fastcgi
	* containerd error: image not found in containerd store: adminer:fastcgi
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/adminer/manifests/fastcgi: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_adminer:fastcgi_2024-06-12_21:10:34.json adminer:fastcgi] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image adminer:fastcgi: failed to pull image adminer:fastcgi: GET https://index.docker.io/v2/library/adminer/manifests/fastcgi: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
adminer:4.8.1-standalone ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_adminer:4.8.1-standalone_2024-06-12_21:10:52.json adminer:4.8.1-standalone] execution failed: exit status 1
Output: 2024-06-12T21:10:53Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:10:54Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "adminer:4.8.1-standalone" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (adminer:4.8.1-standalone): Error response from daemon: No such image: adminer:4.8.1-standalone
	* containerd error: image not found in containerd store: adminer:4.8.1-standalone
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/adminer/manifests/4.8.1-standalone: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_adminer:4.8.1-standalone_2024-06-12_21:10:52.json adminer:4.8.1-standalone] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image adminer:4.8.1-standalone: failed to pull image adminer:4.8.1-standalone: GET https://index.docker.io/v2/library/adminer/manifests/4.8.1-standalone: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
adminer:4.8.1-fastcgi ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_adminer:4.8.1-fastcgi_2024-06-12_21:11:16.json adminer:4.8.1-fastcgi] execution failed: exit status 1
Output: 2024-06-12T21:11:17Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:11:17Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "adminer:4.8.1-fastcgi" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (adminer:4.8.1-fastcgi): Error response from daemon: No such image: adminer:4.8.1-fastcgi
	* containerd error: image not found in containerd store: adminer:4.8.1-fastcgi
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/adminer/manifests/4.8.1-fastcgi: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_adminer:4.8.1-fastcgi_2024-06-12_21:11:16.json adminer:4.8.1-fastcgi] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image adminer:4.8.1-fastcgi: failed to pull image adminer:4.8.1-fastcgi: GET https://index.docker.io/v2/library/adminer/manifests/4.8.1-fastcgi: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
adminer:4.8.1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_adminer:4.8.1_2024-06-12_21:11:39.json adminer:4.8.1] execution failed: exit status 1
Output: 2024-06-12T21:11:39Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:11:40Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "adminer:4.8.1" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (adminer:4.8.1): Error response from daemon: No such image: adminer:4.8.1
	* containerd error: image not found in containerd store: adminer:4.8.1
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/adminer/manifests/4.8.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_adminer:4.8.1_2024-06-12_21:11:39.json adminer:4.8.1] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image adminer:4.8.1: failed to pull image adminer:4.8.1: GET https://index.docker.io/v2/library/adminer/manifests/4.8.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
geonetwork:4.4.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_geonetwork:4.4.4_2024-06-12_21:12:11.json geonetwork:4.4.4] execution failed: exit status 1
Output: 2024-06-12T21:12:12Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:12:12Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "geonetwork:4.4.4" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (geonetwork:4.4.4): Error response from daemon: No such image: geonetwork:4.4.4
	* containerd error: image not found in containerd store: geonetwork:4.4.4
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/geonetwork/manifests/4.4.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_geonetwork:4.4.4_2024-06-12_21:12:11.json geonetwork:4.4.4] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image geonetwork:4.4.4: failed to pull image geonetwork:4.4.4: GET https://index.docker.io/v2/library/geonetwork/manifests/4.4.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
geonetwork:4.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_geonetwork:4.4_2024-06-12_21:13:09.json geonetwork:4.4] execution failed: exit status 1
Output: 2024-06-12T21:13:09Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:13:10Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "geonetwork:4.4" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (geonetwork:4.4): Error response from daemon: No such image: geonetwork:4.4
	* containerd error: image not found in containerd store: geonetwork:4.4
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/geonetwork/manifests/4.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_geonetwork:4.4_2024-06-12_21:13:09.json geonetwork:4.4] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image geonetwork:4.4: failed to pull image geonetwork:4.4: GET https://index.docker.io/v2/library/geonetwork/manifests/4.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
geonetwork:4.2.9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_geonetwork:4.2.9_2024-06-12_21:14:00.json geonetwork:4.2.9] execution failed: exit status 1
Output: 2024-06-12T21:14:00Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:14:01Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "geonetwork:4.2.9" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (geonetwork:4.2.9): Error response from daemon: No such image: geonetwork:4.2.9
	* containerd error: image not found in containerd store: geonetwork:4.2.9
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/geonetwork/manifests/4.2.9: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_geonetwork:4.2.9_2024-06-12_21:14:00.json geonetwork:4.2.9] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image geonetwork:4.2.9: failed to pull image geonetwork:4.2.9: GET https://index.docker.io/v2/library/geonetwork/manifests/4.2.9: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
geonetwork:4.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_geonetwork:4.2_2024-06-12_21:14:56.json geonetwork:4.2] execution failed: exit status 1
Output: 2024-06-12T21:14:57Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:14:57Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "geonetwork:4.2" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (geonetwork:4.2): Error response from daemon: No such image: geonetwork:4.2
	* containerd error: image not found in containerd store: geonetwork:4.2
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/geonetwork/manifests/4.2: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_geonetwork:4.2_2024-06-12_21:14:56.json geonetwork:4.2] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image geonetwork:4.2: failed to pull image geonetwork:4.2: GET https://index.docker.io/v2/library/geonetwork/manifests/4.2: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
geonetwork:4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_geonetwork:4_2024-06-12_21:15:49.json geonetwork:4] execution failed: exit status 1
Output: 2024-06-12T21:15:49Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:15:50Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "geonetwork:4" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (geonetwork:4): Error response from daemon: No such image: geonetwork:4
	* containerd error: image not found in containerd store: geonetwork:4
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/geonetwork/manifests/4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_geonetwork:4_2024-06-12_21:15:49.json geonetwork:4] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image geonetwork:4: failed to pull image geonetwork:4: GET https://index.docker.io/v2/library/geonetwork/manifests/4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
xwiki:stable-postgres-tomcat ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_xwiki:stable-postgres-tomcat_2024-06-12_21:16:39.json xwiki:stable-postgres-tomcat] execution failed: exit status 1
Output: 2024-06-12T21:16:40Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:16:41Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "xwiki:stable-postgres-tomcat" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (xwiki:stable-postgres-tomcat): Error response from daemon: No such image: xwiki:stable-postgres-tomcat
	* containerd error: image not found in containerd store: xwiki:stable-postgres-tomcat
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/xwiki/manifests/stable-postgres-tomcat: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_xwiki:stable-postgres-tomcat_2024-06-12_21:16:39.json xwiki:stable-postgres-tomcat] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image xwiki:stable-postgres-tomcat: failed to pull image xwiki:stable-postgres-tomcat: GET https://index.docker.io/v2/library/xwiki/manifests/stable-postgres-tomcat: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
xwiki:stable-postgres ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_xwiki:stable-postgres_2024-06-12_21:17:46.json xwiki:stable-postgres] execution failed: exit status 1
Output: 2024-06-12T21:17:47Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:17:48Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "xwiki:stable-postgres" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (xwiki:stable-postgres): Error response from daemon: No such image: xwiki:stable-postgres
	* containerd error: image not found in containerd store: xwiki:stable-postgres
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/xwiki/manifests/stable-postgres: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_xwiki:stable-postgres_2024-06-12_21:17:46.json xwiki:stable-postgres] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image xwiki:stable-postgres: failed to pull image xwiki:stable-postgres: GET https://index.docker.io/v2/library/xwiki/manifests/stable-postgres: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
xwiki:stable-mysql-tomcat ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_xwiki:stable-mysql-tomcat_2024-06-12_21:18:49.json xwiki:stable-mysql-tomcat] execution failed: exit status 1
Output: 2024-06-12T21:18:49Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:18:50Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "xwiki:stable-mysql-tomcat" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (xwiki:stable-mysql-tomcat): Error response from daemon: No such image: xwiki:stable-mysql-tomcat
	* containerd error: image not found in containerd store: xwiki:stable-mysql-tomcat
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/xwiki/manifests/stable-mysql-tomcat: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_xwiki:stable-mysql-tomcat_2024-06-12_21:18:49.json xwiki:stable-mysql-tomcat] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image xwiki:stable-mysql-tomcat: failed to pull image xwiki:stable-mysql-tomcat: GET https://index.docker.io/v2/library/xwiki/manifests/stable-mysql-tomcat: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
xwiki:stable-mysql ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_xwiki:stable-mysql_2024-06-12_21:19:56.json xwiki:stable-mysql] execution failed: exit status 1
Output: 2024-06-12T21:19:57Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:19:57Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "xwiki:stable-mysql" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (xwiki:stable-mysql): Error response from daemon: No such image: xwiki:stable-mysql
	* containerd error: image not found in containerd store: xwiki:stable-mysql
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/xwiki/manifests/stable-mysql: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_xwiki:stable-mysql_2024-06-12_21:19:56.json xwiki:stable-mysql] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image xwiki:stable-mysql: failed to pull image xwiki:stable-mysql: GET https://index.docker.io/v2/library/xwiki/manifests/stable-mysql: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
xwiki:stable-mariadb-tomcat ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_xwiki:stable-mariadb-tomcat_2024-06-12_21:20:59.json xwiki:stable-mariadb-tomcat] execution failed: exit status 1
Output: 2024-06-12T21:21:00Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:21:00Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "xwiki:stable-mariadb-tomcat" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (xwiki:stable-mariadb-tomcat): Error response from daemon: No such image: xwiki:stable-mariadb-tomcat
	* containerd error: image not found in containerd store: xwiki:stable-mariadb-tomcat
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/xwiki/manifests/stable-mariadb-tomcat: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_xwiki:stable-mariadb-tomcat_2024-06-12_21:20:59.json xwiki:stable-mariadb-tomcat] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image xwiki:stable-mariadb-tomcat: failed to pull image xwiki:stable-mariadb-tomcat: GET https://index.docker.io/v2/library/xwiki/manifests/stable-mariadb-tomcat: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
groovy:jdk8-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_groovy:jdk8-jammy_2024-06-12_21:22:08.json groovy:jdk8-jammy] execution failed: exit status 1
Output: 2024-06-12T21:22:09Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:22:10Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "groovy:jdk8-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (groovy:jdk8-jammy): Error response from daemon: No such image: groovy:jdk8-jammy
	* containerd error: image not found in containerd store: groovy:jdk8-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/groovy/manifests/jdk8-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_groovy:jdk8-jammy_2024-06-12_21:22:08.json groovy:jdk8-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image groovy:jdk8-jammy: failed to pull image groovy:jdk8-jammy: GET https://index.docker.io/v2/library/groovy/manifests/jdk8-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
groovy:jdk8 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_groovy:jdk8_2024-06-12_21:22:35.json groovy:jdk8] execution failed: exit status 1
Output: 2024-06-12T21:22:35Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:22:36Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "groovy:jdk8" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (groovy:jdk8): Error response from daemon: No such image: groovy:jdk8
	* containerd error: image not found in containerd store: groovy:jdk8
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/groovy/manifests/jdk8: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_groovy:jdk8_2024-06-12_21:22:35.json groovy:jdk8] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image groovy:jdk8: failed to pull image groovy:jdk8: GET https://index.docker.io/v2/library/groovy/manifests/jdk8: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
groovy:jdk21-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_groovy:jdk21-jammy_2024-06-12_21:22:59.json groovy:jdk21-jammy] execution failed: exit status 1
Output: 2024-06-12T21:23:00Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:23:01Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "groovy:jdk21-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (groovy:jdk21-jammy): Error response from daemon: No such image: groovy:jdk21-jammy
	* containerd error: image not found in containerd store: groovy:jdk21-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/groovy/manifests/jdk21-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_groovy:jdk21-jammy_2024-06-12_21:22:59.json groovy:jdk21-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image groovy:jdk21-jammy: failed to pull image groovy:jdk21-jammy: GET https://index.docker.io/v2/library/groovy/manifests/jdk21-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
groovy:jdk21 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_groovy:jdk21_2024-06-12_21:23:26.json groovy:jdk21] execution failed: exit status 1
Output: 2024-06-12T21:23:27Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:23:28Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "groovy:jdk21" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (groovy:jdk21): Error response from daemon: No such image: groovy:jdk21
	* containerd error: image not found in containerd store: groovy:jdk21
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/groovy/manifests/jdk21: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_groovy:jdk21_2024-06-12_21:23:26.json groovy:jdk21] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image groovy:jdk21: failed to pull image groovy:jdk21: GET https://index.docker.io/v2/library/groovy/manifests/jdk21: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
groovy:jdk17-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_groovy:jdk17-jammy_2024-06-12_21:23:52.json groovy:jdk17-jammy] execution failed: exit status 1
Output: 2024-06-12T21:23:53Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:23:54Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "groovy:jdk17-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (groovy:jdk17-jammy): Error response from daemon: No such image: groovy:jdk17-jammy
	* containerd error: image not found in containerd store: groovy:jdk17-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/groovy/manifests/jdk17-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_groovy:jdk17-jammy_2024-06-12_21:23:52.json groovy:jdk17-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image groovy:jdk17-jammy: failed to pull image groovy:jdk17-jammy: GET https://index.docker.io/v2/library/groovy/manifests/jdk17-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
znc:slim ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_znc:slim_2024-06-12_21:24:20.json znc:slim] execution failed: exit status 1
Output: 2024-06-12T21:24:20Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:24:21Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "znc:slim" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (znc:slim): Error response from daemon: No such image: znc:slim
	* containerd error: image not found in containerd store: znc:slim
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/znc/manifests/slim: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_znc:slim_2024-06-12_21:24:20.json znc:slim] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image znc:slim: failed to pull image znc:slim: GET https://index.docker.io/v2/library/znc/manifests/slim: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
znc:1.9.0-slim ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_znc:1.9.0-slim_2024-06-12_21:24:35.json znc:1.9.0-slim] execution failed: exit status 1
Output: 2024-06-12T21:24:36Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:24:37Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "znc:1.9.0-slim" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (znc:1.9.0-slim): Error response from daemon: No such image: znc:1.9.0-slim
	* containerd error: image not found in containerd store: znc:1.9.0-slim
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/znc/manifests/1.9.0-slim: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_znc:1.9.0-slim_2024-06-12_21:24:35.json znc:1.9.0-slim] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image znc:1.9.0-slim: failed to pull image znc:1.9.0-slim: GET https://index.docker.io/v2/library/znc/manifests/1.9.0-slim: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
znc:1.9.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_znc:1.9.0_2024-06-12_21:24:50.json znc:1.9.0] execution failed: exit status 1
Output: 2024-06-12T21:24:51Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:24:52Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "znc:1.9.0" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (znc:1.9.0): Error response from daemon: No such image: znc:1.9.0
	* containerd error: image not found in containerd store: znc:1.9.0
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/znc/manifests/1.9.0: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_znc:1.9.0_2024-06-12_21:24:50.json znc:1.9.0] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image znc:1.9.0: failed to pull image znc:1.9.0: GET https://index.docker.io/v2/library/znc/manifests/1.9.0: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
znc:1.9-slim ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_znc:1.9-slim_2024-06-12_21:25:16.json znc:1.9-slim] execution failed: exit status 1
Output: 2024-06-12T21:25:17Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:25:18Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "znc:1.9-slim" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (znc:1.9-slim): Error response from daemon: No such image: znc:1.9-slim
	* containerd error: image not found in containerd store: znc:1.9-slim
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/znc/manifests/1.9-slim: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_znc:1.9-slim_2024-06-12_21:25:16.json znc:1.9-slim] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image znc:1.9-slim: failed to pull image znc:1.9-slim: GET https://index.docker.io/v2/library/znc/manifests/1.9-slim: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
znc:1.9 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_znc:1.9_2024-06-12_21:25:33.json znc:1.9] execution failed: exit status 1
Output: 2024-06-12T21:25:34Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:25:34Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "znc:1.9" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (znc:1.9): Error response from daemon: No such image: znc:1.9
	* containerd error: image not found in containerd store: znc:1.9
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/znc/manifests/1.9: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_znc:1.9_2024-06-12_21:25:33.json znc:1.9] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image znc:1.9: failed to pull image znc:1.9: GET https://index.docker.io/v2/library/znc/manifests/1.9: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
gradle:jdk-lts-and-current-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_gradle:jdk-lts-and-current-jammy_2024-06-12_21:25:56.json gradle:jdk-lts-and-current-jammy] execution failed: exit status 1
Output: 2024-06-12T21:25:57Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:25:58Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "gradle:jdk-lts-and-current-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (gradle:jdk-lts-and-current-jammy): Error response from daemon: No such image: gradle:jdk-lts-and-current-jammy
	* containerd error: image not found in containerd store: gradle:jdk-lts-and-current-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/gradle/manifests/jdk-lts-and-current-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_gradle:jdk-lts-and-current-jammy_2024-06-12_21:25:56.json gradle:jdk-lts-and-current-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image gradle:jdk-lts-and-current-jammy: failed to pull image gradle:jdk-lts-and-current-jammy: GET https://index.docker.io/v2/library/gradle/manifests/jdk-lts-and-current-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
gradle:jdk-lts-and-current-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_gradle:jdk-lts-and-current-alpine_2024-06-12_21:26:54.json gradle:jdk-lts-and-current-alpine] execution failed: exit status 1
Output: 2024-06-12T21:26:55Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:26:56Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "gradle:jdk-lts-and-current-alpine" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (gradle:jdk-lts-and-current-alpine): Error response from daemon: No such image: gradle:jdk-lts-and-current-alpine
	* containerd error: image not found in containerd store: gradle:jdk-lts-and-current-alpine
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/gradle/manifests/jdk-lts-and-current-alpine: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_gradle:jdk-lts-and-current-alpine_2024-06-12_21:26:54.json gradle:jdk-lts-and-current-alpine] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image gradle:jdk-lts-and-current-alpine: failed to pull image gradle:jdk-lts-and-current-alpine: GET https://index.docker.io/v2/library/gradle/manifests/jdk-lts-and-current-alpine: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
gradle:jdk-lts-and-current ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_gradle:jdk-lts-and-current_2024-06-12_21:27:47.json gradle:jdk-lts-and-current] execution failed: exit status 1
Output: 2024-06-12T21:27:48Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:27:48Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "gradle:jdk-lts-and-current" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (gradle:jdk-lts-and-current): Error response from daemon: No such image: gradle:jdk-lts-and-current
	* containerd error: image not found in containerd store: gradle:jdk-lts-and-current
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/gradle/manifests/jdk-lts-and-current: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_gradle:jdk-lts-and-current_2024-06-12_21:27:47.json gradle:jdk-lts-and-current] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image gradle:jdk-lts-and-current: failed to pull image gradle:jdk-lts-and-current: GET https://index.docker.io/v2/library/gradle/manifests/jdk-lts-and-current: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
gradle:jdk-21-and-22-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_gradle:jdk-21-and-22-jammy_2024-06-12_21:28:40.json gradle:jdk-21-and-22-jammy] execution failed: exit status 1
Output: 2024-06-12T21:28:41Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:28:42Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "gradle:jdk-21-and-22-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (gradle:jdk-21-and-22-jammy): Error response from daemon: No such image: gradle:jdk-21-and-22-jammy
	* containerd error: image not found in containerd store: gradle:jdk-21-and-22-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/gradle/manifests/jdk-21-and-22-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_gradle:jdk-21-and-22-jammy_2024-06-12_21:28:40.json gradle:jdk-21-and-22-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image gradle:jdk-21-and-22-jammy: failed to pull image gradle:jdk-21-and-22-jammy: GET https://index.docker.io/v2/library/gradle/manifests/jdk-21-and-22-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
gradle:jdk-21-and-22-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new8/Trivy_scan_gradle:jdk-21-and-22-alpine_2024-06-12_21:29:34.json gradle:jdk-21-and-22-alpine] execution failed: exit status 1
Output: 2024-06-12T21:29:35Z	INFO	Vulnerability scanning is enabled
2024-06-12T21:29:36Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "gradle:jdk-21-and-22-alpine" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (gradle:jdk-21-and-22-alpine): Error response from daemon: No such image: gradle:jdk-21-and-22-alpine
	* containerd error: image not found in containerd store: gradle:jdk-21-and-22-alpine
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/gradle/manifests/jdk-21-and-22-alpine: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new8/DockerScout_scan_gradle:jdk-21-and-22-alpine_2024-06-12_21:29:34.json gradle:jdk-21-and-22-alpine] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image gradle:jdk-21-and-22-alpine: failed to pull image gradle:jdk-21-and-22-alpine: GET https://index.docker.io/v2/library/gradle/manifests/jdk-21-and-22-alpine: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test gradle:jdk-21-and-22-alpine --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new8/Snyk_scan_gradle:jdk-21-and-22-alpine_2024-06-12_21:29:34.json] execution failed: exit status 2
Output: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
convertigo:8.2.7 ----------- Error pulling image convertigo:8.2.7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
convertigo:8.2 ----------- Error pulling image convertigo:8.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
convertigo:8.2.5 ----------- Error pulling image convertigo:8.2.5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
convertigo:8.2.4 ----------- Error pulling image convertigo:8.2.4: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
convertigo:8.1.0 ----------- Error pulling image convertigo:8.1.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
flink:1.18.1-scala_2.12-java17 ----------- Error pulling image flink:1.18.1-scala_2.12-java17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
flink:1.18.1-scala_2.12-java11 ----------- Error pulling image flink:1.18.1-scala_2.12-java11: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
flink:1.18.1-scala_2.12 ----------- Error pulling image flink:1.18.1-scala_2.12: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
flink:1.18.1-java17 ----------- Error pulling image flink:1.18.1-java17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
flink:1.18.1-java11 ----------- Error pulling image flink:1.18.1-java11: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
swipl:stable ----------- Error pulling image swipl:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
swipl:9.2.5 ----------- Error pulling image swipl:9.2.5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
swipl:9.3.6 ----------- Error pulling image swipl:9.3.6: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
swipl:9.2.4 ----------- Error pulling image swipl:9.2.4: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
swipl:9.3.2 ----------- Error pulling image swipl:9.3.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rust:alpine3.20 ----------- Error pulling image rust:alpine3.20: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rust:alpine ----------- Error pulling image rust:alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rust:1.78.0-alpine3.20 ----------- Error pulling image rust:1.78.0-alpine3.20: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rust:1.78.0-alpine ----------- Error pulling image rust:1.78.0-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rust:1.78-alpine3.20 ----------- Error pulling image rust:1.78-alpine3.20: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
mediawiki:stable-fpm-alpine ----------- Error pulling image mediawiki:stable-fpm-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
mediawiki:stable-fpm ----------- Error pulling image mediawiki:stable-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
mediawiki:stable ----------- Error pulling image mediawiki:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
mediawiki:lts-fpm-alpine ----------- Error pulling image mediawiki:lts-fpm-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
mediawiki:lts-fpm ----------- Error pulling image mediawiki:lts-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sl:7 ----------- Error pulling image sl:7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sl:6 ----------- Error pulling image sl:6: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
open-liberty:kernel-slim-java8-openj9 ----------- Error pulling image open-liberty:kernel-slim-java8-openj9: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
open-liberty:kernel-slim-java17-openj9 ----------- Error pulling image open-liberty:kernel-slim-java17-openj9: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
open-liberty:kernel-slim-java11-openj9 ----------- Error pulling image open-liberty:kernel-slim-java11-openj9: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
open-liberty:kernel-slim ----------- Error pulling image open-liberty:kernel-slim: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
open-liberty:full-java8-openj9 ----------- Error pulling image open-liberty:full-java8-openj9: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
clefos:7.7.1908 ----------- Error pulling image clefos:7.7.1908: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
clefos:7 ----------- Error pulling image clefos:7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
teamspeak:3.13.7 ----------- Error pulling image teamspeak:3.13.7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
teamspeak:3.13 ----------- Error pulling image teamspeak:3.13: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
teamspeak:3.13.6 ----------- Error pulling image teamspeak:3.13.6: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
teamspeak:3.13.5 ----------- Error pulling image teamspeak:3.13.5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
teamspeak:3.13.3 ----------- Error pulling image teamspeak:3.13.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
matomo:fpm ----------- Error pulling image matomo:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
matomo:apache ----------- Error pulling image matomo:apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
matomo:5.0.3-fpm ----------- Error pulling image matomo:5.0.3-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
matomo:5.0.3-apache ----------- Error pulling image matomo:5.0.3-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
matomo:5.0.3 ----------- Error pulling image matomo:5.0.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
yourls:fpm ----------- Error pulling image yourls:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
yourls:apache ----------- Error pulling image yourls:apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
yourls:1.9.2-fpm ----------- Error pulling image yourls:1.9.2-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
yourls:1.9.2-apache ----------- Error pulling image yourls:1.9.2-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
yourls:1.9.2 ----------- Error pulling image yourls:1.9.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
alt:sisyphus ----------- Error pulling image alt:sisyphus: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
alt:p10 ----------- Error pulling image alt:p10: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
alt:p9 ----------- Error pulling image alt:p9: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
alt:p8 ----------- Error pulling image alt:p8: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
amazoncorretto:8u412-al2023-jre ----------- Error pulling image amazoncorretto:8u412-al2023-jre: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
amazoncorretto:8u412-al2023 ----------- Error pulling image amazoncorretto:8u412-al2023: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
amazoncorretto:8u412-al2-native-jre ----------- Error pulling image amazoncorretto:8u412-al2-native-jre: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
amazoncorretto:8u412-al2-native-jdk ----------- Error pulling image amazoncorretto:8u412-al2-native-jdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
amazoncorretto:8u412-al2-generic ----------- Error pulling image amazoncorretto:8u412-al2-generic: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
postfixadmin:fpm ----------- Error pulling image postfixadmin:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
postfixadmin:apache ----------- Error pulling image postfixadmin:apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
postfixadmin:3.3.13-fpm ----------- Error pulling image postfixadmin:3.3.13-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
postfixadmin:3.3.13-apache ----------- Error pulling image postfixadmin:3.3.13-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
postfixadmin:3.3.13 ----------- Error pulling image postfixadmin:3.3.13: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
fluentd:v1.16.2-debian-1.1 ----------- Error pulling image fluentd:v1.16.2-debian-1.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
fluentd:v1.16-debian-1 ----------- Error pulling image fluentd:v1.16-debian-1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
fluentd:v1.16.2-1.1 ----------- Error pulling image fluentd:v1.16.2-1.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
fluentd:v1.16-1 ----------- Error pulling image fluentd:v1.16-1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
fluentd:v1.16.0-debian-1.0 ----------- Error pulling image fluentd:v1.16.0-debian-1.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sapmachine:ubuntu-noble ----------- Error pulling image sapmachine:ubuntu-noble: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sapmachine:ubuntu-jammy ----------- Error pulling image sapmachine:ubuntu-jammy: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sapmachine:ubuntu-focal ----------- Error pulling image sapmachine:ubuntu-focal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sapmachine:ubuntu-24.04 ----------- Error pulling image sapmachine:ubuntu-24.04: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
sapmachine:ubuntu-22.04 ----------- Error pulling image sapmachine:ubuntu-22.04: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:stable ----------- Error pulling image varnish:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:old ----------- Error pulling image varnish:old: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:fresh ----------- Error pulling image varnish:fresh: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:7.5.0 ----------- Error pulling image varnish:7.5.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
varnish:7.5 ----------- Error pulling image varnish:7.5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:multilib-devel-20240101.0.204074 ----------- Error pulling image archlinux:multilib-devel-20240101.0.204074: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:multilib-devel ----------- Error pulling image archlinux:multilib-devel: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:base-devel-20240101.0.204074 ----------- Error pulling image archlinux:base-devel-20240101.0.204074: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:base-devel ----------- Error pulling image archlinux:base-devel: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
archlinux:base-20240101.0.204074 ----------- Error pulling image archlinux:base-20240101.0.204074: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:stable-fpm ----------- Error pulling image friendica:stable-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:stable-apache ----------- Error pulling image friendica:stable-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:stable ----------- Error pulling image friendica:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:fpm ----------- Error pulling image friendica:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
friendica:dev-fpm ----------- Error pulling image friendica:dev-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:builder-alpine ----------- Error pulling image caddy:builder-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:builder ----------- Error pulling image caddy:builder: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:alpine ----------- Error pulling image caddy:alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:2.8.4-builder-alpine ----------- Error pulling image caddy:2.8.4-builder-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
caddy:2.8.4-builder ----------- Error pulling image caddy:2.8.4-builder: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta.4-fpm ----------- Error pulling image monica:5.0.0-beta.4-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta.4-apache ----------- Error pulling image monica:5.0.0-beta.4-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta-fpm ----------- Error pulling image monica:5.0.0-beta-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0.0-beta-apache ----------- Error pulling image monica:5.0.0-beta-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
monica:5.0-fpm ----------- Error pulling image monica:5.0-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:fpm ----------- Error pulling image phpmyadmin:fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:apache ----------- Error pulling image phpmyadmin:apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:5.2.1-fpm ----------- Error pulling image phpmyadmin:5.2.1-fpm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:5.2.1-apache ----------- Error pulling image phpmyadmin:5.2.1-apache: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
phpmyadmin:5.2.1 ----------- Error pulling image phpmyadmin:5.2.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.8.0-1 ----------- Error pulling image hitch:1.8.0-1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.8.0 ----------- Error pulling image hitch:1.8.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.8 ----------- Error pulling image hitch:1.8: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1 ----------- Error pulling image hitch:1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
hitch:1.7.3-1 ----------- Error pulling image hitch:1.7.3-1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:minimal ----------- Error pulling image almalinux:minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4-minimal-20240530 ----------- Error pulling image almalinux:9.4-minimal-20240530: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4-minimal ----------- Error pulling image almalinux:9.4-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4-20240530 ----------- Error pulling image almalinux:9.4-20240530: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
almalinux:9.4 ----------- Error pulling image almalinux:9.4: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:stable-sdk ----------- Error pulling image dart:stable-sdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:stable ----------- Error pulling image dart:stable: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:sdk ----------- Error pulling image dart:sdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:beta-sdk ----------- Error pulling image dart:beta-sdk: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
dart:beta ----------- Error pulling image dart:beta: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8u412-b08-jre-ubi9-minimal ----------- Error pulling image eclipse-temurin:8u412-b08-jre-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8u412-b08-jdk-ubi9-minimal ----------- Error pulling image eclipse-temurin:8u412-b08-jdk-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8-ubi9-minimal ----------- Error pulling image eclipse-temurin:8-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8-jre-ubi9-minimal ----------- Error pulling image eclipse-temurin:8-jre-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
eclipse-temurin:8-jdk-ubi9-minimal ----------- Error pulling image eclipse-temurin:8-jdk-ubi9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jre-jammy ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jre-jammy: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jre-focal ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jre-focal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jre ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jre: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jdk-jammy ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jdk-jammy: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
ibm-semeru-runtimes:open-8u412-b08-jdk-focal ----------- Error pulling image ibm-semeru-runtimes:open-8u412-b08-jdk-focal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3.20231119-minimal ----------- Error pulling image rockylinux:9.3.20231119-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3.20231119 ----------- Error pulling image rockylinux:9.3.20231119: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3-minimal ----------- Error pulling image rockylinux:9.3-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9.3 ----------- Error pulling image rockylinux:9.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
rockylinux:9-minimal ----------- Error pulling image rockylinux:9-minimal: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.3 ----------- Error pulling image api-firewall:0.7.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.2 ----------- Error pulling image api-firewall:0.7.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.1 ----------- Error pulling image api-firewall:0.7.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.7.0 ----------- Error pulling image api-firewall:0.7.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.6.17 ----------- Error pulling image api-firewall:0.6.17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.7.0 ----------- Error pulling image emqx:5.7.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.7 ----------- Error pulling image emqx:5.7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5 ----------- Error pulling image emqx:5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.6.1 ----------- Error pulling image emqx:5.6.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.6 ----------- Error pulling image emqx:5.6: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:bookworm ----------- Error pulling image satosa:bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4.0-bookworm ----------- Error pulling image satosa:8.4.0-bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4.0 ----------- Error pulling image satosa:8.4.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4-bookworm ----------- Error pulling image satosa:8.4-bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4 ----------- Error pulling image satosa:8.4: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8.3 ----------- Error pulling image unit:php8.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8.2 ----------- Error pulling image unit:php8.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8 ----------- Error pulling image unit:php8: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php ----------- Error pulling image unit:php: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:1.32.1-php8.3 ----------- Error pulling image unit:1.32.1-php8.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:scala ----------- Error pulling image spark:scala: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:r ----------- Error pulling image spark:r: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:python3-java17 ----------- Error pulling image spark:python3-java17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:python3 ----------- Error pulling image spark:python3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:3.5.1-scala2.12-java17-ubuntu ----------- Error pulling image spark:3.5.1-scala2.12-java17-ubuntu: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28.0 ----------- Error pulling image liquibase:4.28.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28 ----------- Error pulling image liquibase:4.28: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:alpine ----------- Error pulling image liquibase:alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28.0-alpine ----------- Error pulling image liquibase:4.28.0-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28-alpine ----------- Error pulling image liquibase:4.28-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
